common/src/main/java/com/wynntils/features/debug/PacketDebuggerFeature.java

```diff
@@ -83,6 +86,7 @@ private String describePacket(Packet<?> packet) {
     public void onPacketSent(PacketSentEvent<?> e) {
         Packet<?> packet = e.getPacket();
         if (IGNORE_LIST.contains(packet.getClass())) return;
+        if (packet.getClass() != ServerboundContainerClickPacket.class) return;
```

Comment:
Left over debug code..?

Reply:
Yes, committed until I worked on this. I'll keep this open for now.